testdata/p4_16_samples/spec-ex19.p4

```diff
@@ -53,7 +53,8 @@ parser Tcp_option_parser(packet_in b,
     }
     state parse_tcp_option_sack {
          b.extract(vec.next.sack,
-                   (bit<32>)(b.lookahead<Tcp_option_sack_top>().length));
+                   (bit<32>)(8 * (b.lookahead<Tcp_option_sack_top>().length) -
```

Comment:
For testing the compiler this is fine. For production code this would need to be more complicated.
For the spec we try to make a point about a language feature.
Let's leave this test as you wrote it.